src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.scss

```diff
@@ -1,30 +1,22 @@
-@use './src/styles/vendor/variables' as vv;
-
 .container-fluid {
   align-items: flex-start;
   display: flex;
   padding-left: 0;
   width: 100%;
 }
 
-.card-body {
-  max-width: 85%;
-}
-
-.vertical-line {
-  border-left: 1px solid vv.$gray-400;
-}
-
-cd-wizard {
-  width: 15%;
-}
-
 cd-hosts {
   ::ng-deep .nav {
     display: none;
   }
 }
 
-.alignForm {
-  margin-left: -1%;
+cd-osd-form {
+  ::ng-deep .card {
+    border: 0;
+  }
+
+  ::ng-deep .accordion {
+    margin-left: -1.5rem;
+  }
```

Comment:
Rather that styling here (with the help of the forever-to-be-deprecated `::ng-deep`), could we use global styles? Is there any reason why we want a card or accordion classes to behave differently here (IMHO something that Angular fosters too easily). We should avoid as much as we can per-component styling, and favor global-consistent styling.

Sometimes the reason for having to style per component is that an existing per-component styling is interfering with us (specially with `::ng-deep` cause it traverses levels). The more we add here, the more often we'll have to add per-component style.

Reply:
I've tried to use the global styling mostly but for the cluster expansion this is needed unfortunately. This page needs a different styling so that it doesn't look weird. Otherwise it feels like its not aligned properly. Hence the per-styling component. Apart from this I tried to avoid per component styling mostly and used the global ones.

Reply:
Ok.